src/components/StudyPane/InfoPane/Motif/RootBlock_old.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState, useEffect, useContext } from 'react';
```

Comment:
What is this old file for? You can get rid of it as we can get the old version from history